core/src/main/scala/kafka/cluster/Replica.scala

```diff
@@ -71,7 +75,8 @@ class Replica(val brokerId: Int, val topicPartition: TopicPartition) extends Log
   def updateFetchState(followerFetchOffsetMetadata: LogOffsetMetadata,
                        followerStartOffset: Long,
                        followerFetchTimeMs: Long,
-                       leaderEndOffset: Long): Unit = {
+                       leaderEndOffset: Long,
+                       isAddingReplica: Boolean): Unit = {
```

Comment:
`isAddingReplica` is unused

Reply:
Thanks, even IntelliJ didn't notice it :D